api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -56,7 +56,17 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
               .commercialPurpose(workspace.getCommercialPurpose())
               .population(workspace.getPopulation())
               .populationOfFocus(workspace.getPopulationOfFocus())
-              .additionalNotes(workspace.getAdditionalNotes());
+              .additionalNotes(workspace.getAdditionalNotes())
```

Comment:
Could you add a test of round trip between db and Swagger model? (Maybe verifying that the Swagger serialization is identical after a round trip.) My guess is that at some point we'll mess up the conversion going one way or the other.

Reply:
I believe the testCreateWorkspace already does this round trip test, but I'll add some detail to that to cover the new fields.